internal/api/v1beta1/user.go

```diff
@@ -90,6 +92,12 @@ func (h Handler) CreateUser(ctx context.Context, request *shieldv1beta1.CreateUs
 		email = currentUserEmail
 	}
 
+	name := request.GetBody().GetName()
+	slug := strings.TrimSpace(request.GetBody().GetSlug())
+	if slug == "" {
+		slug = str.GenerateUserSlug(name)
```

Comment:
Use email instead

Reply:
yes will update the slug generation strategy